cfg/config_test.go

```diff
@@ -44,6 +46,7 @@ func TestEnvVars(t *testing.T) {
 				"NAMESPACE":           "my-namespace",
 				"NODE_SELECTOR":       "{\"type\": \"compute\"}",
 				"CACHING_CPU_REQUEST": ".055",
+				"NODE_TOLERATION":"[{\"effect\": \"NoSchedule\",\"key\": \"app\",\"operator\": \"Equal\",\"value\": \"prod\"}]",
```

Comment:
Consider using a raw string here:
```suggestion
				"NODE_TOLERATION":`[{"effect": "NoSchedule","key": "app","operator": "Equal","value": "prod"}]`,
```